lib/fog/aws/parsers/compute/describe_vpcs.rb

```diff
@@ -24,22 +28,33 @@ def end_element(name)
               case name
                 when 'item'
                   @vpc['tagSet'][@tag['key']] = @tag['value']
-                  @tag = {}
```

Comment:
Why is this removed? Seems like it would be unrelated to the cidr block sets and might break stuff.